community/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/SemanticErrorAcceptanceTest.scala

```diff
@@ -282,15 +282,15 @@ class SemanticErrorAcceptanceTest extends ExecutionEngineFunSuite {
     )
   }
 
-  test("should fail if using a hint with property equality comparison") {
+  test("should fail if using a hint with multiple OR'd equality comparisons") {
     executeAndEnsureError(
-      "match (n:Person)-->(m:Person) using index n:Person(name) where n.name = m.name return n",
+      "match (n:Person) using index n:Person(name) where n.name = 'a' OR n.name = 'b' return n",
```

Comment:
Why was this changed? Are we now supporting queries on the form that was replaced here?